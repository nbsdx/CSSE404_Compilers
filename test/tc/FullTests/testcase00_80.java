class EntryPoint {
	public static void main(String[] args) {
		Point p = new Point();
		Point q = new Point();
	}
}

class Point extends DoesNotExist {}

class Point {}

class Point {}