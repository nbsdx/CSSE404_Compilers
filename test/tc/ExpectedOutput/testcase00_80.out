Superclass name DoesNotExist not in scope.
Class named Point already exists.
Class named Point already exists.
